packages/sysutils/busybox/scripts/init

```diff
@@ -719,7 +719,6 @@
 
           mount -o remount,rw /flash
           sed -i "s/^DEFAULT .*/DEFAULT $SYSLINUX_DEFAULT/" /flash/syslinux.cfg
-          [ -f /flash/EFI/BOOT/syslinux.cfg ] && cp /flash/syslinux.cfg /flash/EFI/BOOT/syslinux.cfg
```

Comment:
The line is still needed if a LE8 system is updated to LE9. 

Or remove any /flash/EFI/BOOT/syslinux.cfg here.

Reply:
Ah yes, you're right.

This code is only used by the `installer`/`live`/`run` modes (typically, when booting from a USB memory stick) and we won't have included `EFI/BOOT/syslinux.cfg` in such a build.

However, if someone is booting their PC via USB memory stick (or has dd'd the img directly to HDD disk) and is always using `run` mode to boot, then once they upgrade from LE8 the new LE9 `init` will stop overwriting their EFI config, and they would "lose" the ability to remember the last boot mode (assuming they ever booted into another mode).

I'll update the commit so that it removes `/flash/EFI/BOOT/syslinux.cfg` - thanks.